packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -213,8 +218,67 @@ export default declare((api, opts) => {
       // function a({ b, ...c }) {}
       Function(path) {
         const params = path.get("params");
-        for (let i = params.length - 1; i >= 0; i--) {
-          replaceRestElement(params[i].parentPath, params[i]);
+        const paramHasRestElement = [];
```

Comment:
It suffices to model `paramHasRestElement` as `Set<number>` and to only include the index of which has rest element.

Reply:
Changed. I used an array of booleans instead of a set since the code was about the same length and I figured array access would be (very marginally) faster than set access.